stable/kiam/templates/agent-daemonset.yaml

```diff
@@ -85,7 +85,7 @@ spec:
             - --cert=/etc/kiam/tls/{{ .Values.agent.tlsCerts.certFileName }}
             - --key=/etc/kiam/tls/{{ .Values.agent.tlsCerts.keyFileName }}
             - --ca=/etc/kiam/tls/{{ .Values.agent.tlsCerts.caFileName }}
-            - --server-address={{ template "kiam.fullname" . }}-server:{{ .Values.server.service.port }}
+            - --server-address={{ if .Values.agent.serverHostOverride }}{{ .Values.agent.serverHostOverride }}{{ else if .Values.server.useHostNetwork }}localhost{{ else }}{{ template "kiam.fullname" . }}-server{{ end }}:{{ .Values.server.service.port }}
```

Comment:
Can you split over multiple lines? A single line gets unreadable.
e.g.
```
{{ if .Values.agent.serverHostOverride }}
- --server-address={{ .Values.agent.serverHostOverride }}
{{ else if .Values.server.useHostNetwork }}
...
```

Reply:
Fixed.